chatGPT_API/Model/QueryManager.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  QueryManager.swift
+//  chatGPT_API
+//
+//  Created by Ai Hawok on 19/03/2023.
+//
+// QueryManager consists of two queries. Opening and Continue.
+// queryOpening: set's up chatGPT to play text based story game;
+// queryContinue: continues the story based on the choice user made
+import Foundation
+
+class QueryManager{
+
+    var queryOpening =
+    """
+    Lets play text based story game, Generate me a short story and three short options to choose from. You should return me json with following structure. For now create only beginning of the story and don't continue until I choose an option
+    {
+        "story" : "String",
+        "options" : ["Option 1", "Option 2", "Option 3"]
+     }
```

Comment:
Given you're always going to get three choices, you could make them be given to you as three unique keys:
```swift
{
  "story": "String",
  "choice1": "String",
  "choice2": "String",
  "choice3": "String"
}
```
Then you could make `Story` Decodable and part of `Message`:

```swift
struct Message: Decodable{
    let content: Story
}
```
And this removes the need for `StoryBrain`.